### ISSUE TITLE
Can not connect to substrate front end template  API/INIT: Error: FATAL: Unable to initialize the API: Cannot convert metadata from version 14 to 13

### ISSUE THREAD

**Issue by arenouf14** [substrate-developer-hub/substrate-front-end-template#199]
After having launch the substrate-node-template and blocks are produced, I start the substrate-frontend-template with yarn install then I try to connect to localhost:8000 and I get Error Connecting to Substrate [isTruster:true].
I can not connect to the interface.

In Firefox Developer Tools Console:

[HMR] Waiting for update signal from WDS... log.js:24
Connected socket: ws://127.0.0.1:9944 SubstrateContext.js:14
web3Enable: Enabled 0 extensions: index.js:88
web3Accounts: Found 0 addresses: index.js:121

2021-10-02 02:59:48        API/INIT: Error: FATAL: Unable to initialize the API: Cannot convert metadata from version 14 to 13
    value Init.js:68
index.js:1
    e index.js:1
    apply logger.js:47
    error logger.js:96
    value Init.js:69

**Comment by jaredcbaum** [comment]
Possible related to #197

**Comment by jaredcbaum** [comment]
It works for a split second before crashing after updating `package.json` dependencies `@polkadot/api` and `@polkadot/types` to latest (v6.1.2 atm). I tried updating `@polkadot` deps to latest as well, but still crashes.

**Comment by arenouf14** [comment]
> It works for a split second before crashing after updating `package.json` dependencies `@polkadot/api` and `@polkadot/types` to latest (v6.1.2 atm). I tried updating all other `@polkadot` deps to latest as well, but still crashes.

Exactly same issue, I can see the ids of 5 accounts be displayed in Console then crash

**Comment by jimmychu0807** [comment]
Close by #200

**Comment by arenouf14** [comment]
I still have the same problem (same error message) even after having pulled the changes.
I tried to use the node from substrate-monthly-2021-08 and run it. With this node version it works and I have Metadata v13 that corresponds to what is waited for.

**Comment by jaredcbaum** [comment]
At first glance, works for me after these updates. Will update if I have problems down the line. Thanks for the timely fix!

**Comment by jimmychu0807** [comment]
@arenouf14 Are you using [Substrate Node Template `latest` version](https://github.com/substrate-developer-hub/substrate-node-template/tree/latest) with [Substrate Front-end Template `latest` version](https://github.com/substrate-developer-hub/substrate-front-end-template/tree/latest) together? After you check out the front end tpl, be sure to run `yarn install` first. Does it work for you?

Both node template and front-end template should have the same version (or monthly tag) when using together. This is also how we test them here.

**Comment by arenouf14** [comment]
@jimmychu0807 Just wanted to add that in order for it to work well when you open a SSH tunnel to forward the port 8000 on localhost, we need --ws-external argument to node-template (I did not see in the documentation but looking for why it does not work anymore last night...)

**Comment by godhar** [comment]
I am having a similar issue in my console when following the kitties tutorial. I run the backend, which starts fine. But the React app only shows the spinner. This is in the console:
 
`Connected socket: ws://127.0.0.1:9944
bundle.js:88 web3Enable: Enabled 0 extensions: 
bundle.js:124 web3Accounts: Found 0 addresses: `

I believe it should pick up the default accounts, Alice, Bob etc. Am out of ideas of how to troubleshoot this, as I said the `node-template` has been running ok.

front-end template `"version": "monthly-2022-01"`
node-template `version = '4.0.0-dev'`

**Comment by jimmychu0807** [comment]
@godhar 

1. How do you compile and run the front-end template?
2. Regarding Node Template, have you tried with [`monthly-2021-12` tag of Node Template](https://github.com/substrate-developer-hub/substrate-node-template/tree/monthly-2021-12)?

**Comment by godhar** [comment]
@jimmychu0807

1. Using `yarn install` then `yarn start`
2. I tried now `node-template` of `monthly-2021-12` you suggested, but the same issue running that backend.

I noticed that `allAccounts`is an empty array, in `SubstrateContext.js` - if this helps reveal something maybe.

`const loadAccounts = (state, dispatch) => {
  const asyncLoadAccounts = async () => {
    dispatch({ type: 'LOAD_KEYRING' })
    try {
      await web3Enable(config.APP_NAME)
      let allAccounts = await web3Accounts()
      console.log(allAccounts)
      ...`
      
      Anything else I can check?
